ml_enabler/tests/test_utils.py

```diff
@@ -0,0 +1,110 @@
+from ml_enabler.models.utils import geojson_bounds, polygon_to_wkt
+from ml_enabler.tests.base import BaseTestCase
+
+
```

Comment:
slight nitpicking here, but can we simplify the lat/lng values used in the tests? like just truncate to two decimals or so - would just make the fixture easier to read through and easier to modify down the line if needed.